esbuild/esbuild.config.js

```diff
@@ -36,10 +36,6 @@ const buildConfig = {
         from: path.join(cwd, 'assets/images/**/*'),
         to: path.join(cwd, 'dist/assets/images'),
       },
-      {
-        from: path.join(cwd, 'assets/js/**/*'),
```

Comment:
don't now need to copy all javascript files over to assets, they'll get automatically bundled instead